addons/mrp/models/mrp_production.py

```diff
@@ -649,6 +649,10 @@ def _get_moves_raw_values(self):
 
     def _get_move_raw_values(self, product_id, product_uom_qty, product_uom, operation_id=False, bom_line=False):
```

Comment:
``` python
def _get_move_raw_values(self, product_id, product_uom_qty, product_uom, operation_id=False, bom_line=False):
    """Called when the quantity to produce is changed on a manufacturing order and a new raw
    move needs to be created.
    """
```

also why does this method have so many args and kwargs but is used exactly one time ?..

Reply:
wow this code is so unreadable

_get_move_raw_values
_get_moves_raw_values
_update_raw_move
wth

Reply:
> ```python
> def _get_move_raw_values(self, product_id, product_uom_qty, product_uom, operation_id=False, bom_line=False):
>     """Called when the quantity to produce is changed on a manufacturing order and a new raw
>     move needs to be created.
>     """
> ```
> 
> also why does this method have so many args and kwargs but is used exactly one time ?..

I change the signature in the 'flexible' task to call the method without bom line data